cmd/tfplugindocs/testdata/scripts/generate/framework_provider_success_generic_templates.txtar

```diff
@@ -0,0 +1,928 @@
+# Successful run of tfplugindocs on a Framework provider with "generic" template paths (i.e. templates/resources.md.tmpl)
+# Templates test all implemented data fields and functions.
+[!unix] skip
+env GOCACHE=$WORK/gocache
+env GOMODCACHE=$WORK/gomodcache
+exec tfplugindocs --provider-name=terraform-provider-scaffolding
+cmp stdout expected-output.txt
+cmpenv docs/index.md expected-index.md
+cmpenv docs/data-sources/example.md expected-datasource.md
+cmpenv docs/resources/example.md expected-resource.md
```

Comment:
Is `testscript` able to error if there are extra (unexpected) files?

Reply:
Yes, but it wouldn't be straightforward. We can call the shell to use the `ls` command on the directory that we want and pipe the results to a file then do a golden file comparison.

Reply:
Bummer -- probably not worth the hassle trying to jam that in then.